src/recognizers_store.proto

```diff
@@ -8,11 +8,17 @@ message PatternRecognizer {
     // The Regex used by this recognizer
     string pattern = 2;
     // The confidence of the result
-    float strength = 3;
+    float score = 3;
     // The entity name which this recognizer can detect
     string entity = 4;
-    // The supported language
+    // The supported language code, in ISO-639 format, https://en.wikipedia.org/wiki/ISO_639-1
     string language = 5;
```

Comment:
@itye-msft please make sure you fix languageCode to language in the API changes you made, to align with @ELad's changes.